app/controllers/interviews_controller.rb

```diff
@@ -0,0 +1,47 @@
+class InterviewsController < ApplicationController
+  #skip_before_action :authenticate_user!, only: [:index, :show]
+
+  def index
+    # add Search!
+    @interviews = Interview.all
+  end
+
+  def show
+    find_interview
+    #authorize @interview
+    @user = current_user
+  end
+
+  def new
+    @interview = Interview.new
+    #authorize @interview
+  end
+
+  def create
+    @interview = Interview.new(interview_params)
+    #authorize @interview
+    @interview.user = current_user
+    if @interview.save
+      redirect_to interview_path(@interview)
```

Comment:
I believe we would redirect to the dashboard, not the interview :show view.

Reply:
true